src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,10 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	/**
+	 * @return the feedbackToUser
```

Comment:
For simple method, it's better to use method description than `@return`. Refer to #188